<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FavouriteTools</class>
 <widget class="QDialog" name="FavouriteTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Favourites</string>
  </property>
  <property name="windowIcon">
   <iconset theme="star" resource="../../resource.qrc">
    <normaloff>:/Default/icons/star.png</normaloff>:/Default/icons/star.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFavouriteTool">
      <attribute name="title">
       <string>Tools</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="listTools"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddFavouriteTool">
           <property name="text">
            <string>&amp;Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEditFavouriteTool">
           <property name="text">
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveFavouriteTool">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDescriptionFavouriteTool">
           <property name="text">
            <string>De&amp;scription...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpFavouriteTool">
           <property name="text">
            <string>&amp;Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDownFavouriteTool">
           <property name="text">
            <string>&amp;Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFavouriteScript">
      <attribute name="title">
       <string>Script</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="listScript"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAddFavouriteScript">
           <property name="text">
            <string>&amp;Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEditFavouriteScript">
           <property name="text">
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveFavouriteScript">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDescriptionFavouriteScript">
           <property name="text">
            <string>De&amp;scription...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpFavouriteScript">
           <property name="text">
            <string>&amp;Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDownFavouriteScript">
           <property name="text">
            <string>&amp;Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFavouriteCommand">
      <attribute name="title">
       <string>Command</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="listCommand"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btnAddFavouriteCommand">
           <property name="text">
            <string>&amp;Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEditFavouriteCommand">
           <property name="text">
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveFavouriteCommand">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpFavouriteCommand">
           <property name="text">
            <string>&amp;Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDownFavouriteCommand">
           <property name="text">
            <string>&amp;Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Shortcut</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="ShortcutEdit" name="shortcutEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearShortcut">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutEdit</class>
   <extends>QLineEdit</extends>
   <header>ShortcutEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>FavouriteTools</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
